author: Andres Salomon <dilinger@debian.org>

Work around the following with libstdc++ 12:

./../components/autofill/core/browser/data_model/autofill_i18n_api.h:18:30: error: constexpr variable 'kLegacyHierarchyCountryCodeString' must be initialized by a constant expression
constexpr inline std::string kLegacyHierarchyCountryCodeString{"XX"};
                             ^~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~

Index: chromium-120.0.6099.71/components/autofill/core/browser/data_model/autofill_i18n_api.h
===================================================================
--- chromium-120.0.6099.71.orig/components/autofill/core/browser/data_model/autofill_i18n_api.h
+++ chromium-120.0.6099.71/components/autofill/core/browser/data_model/autofill_i18n_api.h
@@ -15,8 +15,8 @@ namespace autofill::i18n_model_definitio
 // Country code that represents autofill's legacy address hierarchy model as
 // stored `kAutofillModelRules`. As a workaround for GCC we declare the
 // std::string constexpr first.
-constexpr inline std::string kLegacyHierarchyCountryCodeString{"XX"};
-constexpr AddressCountryCode kLegacyHierarchyCountryCode =
+const inline std::string kLegacyHierarchyCountryCodeString{"XX"};
+const AddressCountryCode kLegacyHierarchyCountryCode =
     AddressCountryCode(kLegacyHierarchyCountryCodeString);
 
 // Creates an instance of the address hierarchy model corresponding to the
